lib/internal/per_context/primordials.js

```diff
@@ -174,5 +174,8 @@ primordials.SafePromise = makeSafe(
   copyPrototype(original.prototype, primordials, `${name}Prototype`);
 });
 
+primordials.TypedArrayPrototypeSubarray =
+  uncurryThis(Uint8Array.prototype.subarray);
```

Comment:
Isn't it the same as `primordials.Uint8ArrayPrototypeSubarray` ? If you want to keep it because of the more generic name, you could also copy the other methods with `copyPrototype(Uint8Array.prototype, primordials, `TypedArrayPrototype`)`

Reply:
No, I’m getting `Uint8ArrayPrototypeSubarray` Is not a function. The reason seems to be https://github.com/nodejs/node/pull/36003#discussion_r518916389.

Reply:
I’m adding `TypedArrayPrototype` to `primordials` in <https://github.com/nodejs/node/pull/36016>.